build/stage-build.yml

```diff
@@ -11,7 +11,7 @@
   displayName: 'Calculate version'
 
 - task: DotNetCoreCLI@2
-  displayName: 'Install .Net 6 workloads'
+  displayName: 'Install .NET 8 Workloads'
```

Comment:
How do you know it's actually .NET 8?

Reply:
I used the way we use on Chinook instead.